src/Microsoft.AspNet.Mvc.Formatters.Xml/MvcXmlDataContractSerializerMvcOptionsSetup.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.Formatters.Xml;
+using Microsoft.AspNet.Mvc.ModelBinding.Metadata;
+using Microsoft.Framework.OptionsModel;
+
+namespace Microsoft.AspNet.Mvc
+{
```

Comment:
docs for the class and `ConfigureMvc` method. Same for `MvcXmlSerializerMvcOptionsSetup` class too.